pyccel/codegen/printing/ccode.py

```diff
@@ -773,7 +746,9 @@ def _print_PythonMinMax(self, expr):
         elif len(arg) > 2 and isinstance(arg.dtype.primitive_type, (PrimitiveFloatingPointType, PrimitiveIntegerType)):
             key = self.get_declare_type(arg[0])
             self.add_import(Import('stc/common', AsName(VariableTypeAnnotation(arg.dtype), key)))
-            self.add_import(Import('Common_extensions', AsName(VariableTypeAnnotation(arg.dtype), key)))
+            self.add_import(Import('Common_extensions',
+                                   AsName(VariableTypeAnnotation(arg.dtype), key),
+                                   ignore_at_print=True))
```

Comment:
Same question below

Reply:
Yes it is needed so that `manage_dependencies` in `pyccel/codegen/utilities.py` can include the internal lib:
```python
  for lib_name, (stdlib_folder, stdlib) in internal_libs.items():
        if lib_name in printer.get_additional_imports():

            lib_dest_path = copy_internal_library(stdlib_folder, pyccel_dirpath)
```
We can, of course, remove the `self.add_import` and defer that logic to `manage_dependencies` and automatically include the extension when treating the respective stc import. What do you think  is best?

Reply:
Hmm, this may be related to #2064 then. I am struggling to manage my imports there